### PR TITLE
CDN publishing

### PR DESCRIPTION
**Pull request by joseluisdiaz**
Remove CDN purge to support s3 cloudfront max-age cache control instead

### CODE REVIEW

File: scripts/upload.sh [hunk 1]
```diff
@@ -35,13 +35,7 @@ s3_upload()
     echo ""
     verbose_item "Uploading '$1'"
     # aws s3 cp "$DRY_RUN" --region us-west-1 "$1" "$REMOTE_PATH/$1"
```
Comment (dafortune) [inline scripts/upload.sh]: I think this comment does not makes sense anymore; do you know what it used to do?

File: scripts/upload.sh [hunk 2]
```diff
@@ -35,13 +35,7 @@ s3_upload()
...
-    aws s3 cp "$1" "$REMOTE_PATH/$1" --region us-west-1
-    if [ -z "$DRY_RUN" ]
-    then
-      curl -fs -XDELETE "https://cdn.auth0.com/js/$LIBRARY_NAME/$VERSION/$1"
-    else
-      echo "(dryrun) curl -fs -XDELETE https://cdn.auth0.com/js/$LIBRARY_NAME/$VERSION/$1"
-    fi
+    aws s3 cp "$1" "$REMOTE_PATH/$1" --region us-west-1 --cache-control "public, max-age=84600"
```
Comment (dafortune) [inline scripts/upload.sh]: What about version vs. wildcard version (https://github.com/auth0/auth0-guardian.js/blob/master/scripts/ci.sh#L98) vs https://github.com/auth0/auth0-guardian.js/blob/master/scripts/ci.sh#L95 they will have the same cache time (23.5h) I'm fine with that, but was that what you intended?

Comment (joseluisdiaz) [inline scripts/upload.sh]: No; good catch. But im going to use only one in the mfa-widget too; so i think that is ok for now.

File: scripts/upload.sh (hunk 1 above)
Comment (joseluisdiaz) [inline scripts/upload.sh]: debugging? i guess... it's better to put an `echo` before the command to see the result. Will remove it.

**Pull request merged**